### PR TITLE
[Serve] Fix Formatting, stale docs

### PR DESCRIPTION
**Pull request by bllchmbrs**
The documentation was incorrect.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [X] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]
LGTM

File: doc/source/serve/advanced.rst [hunk 1]
```diff
@@ -269,3 +269,35 @@ Once a endpoint is deleted, its tag can be reused.
 .. code-block:: python
 
   serve.delete_endpoint("simple_endpoint")
+
+How do I get a reference to a "backend" class?
```
Comment (edoakes) [inline doc/source/serve/advanced.rst:273]: "reference" isn't very clear to me - what about "how to I call an endpoint from within my backend?"

Comment (edoakes) [inline doc/source/serve/advanced.rst:273]: This is actually a reference to an _endpoint_, btw, not a backend

**Review by edoakes — CHANGES_REQUESTED** [review]

File: doc/source/serve/advanced.rst [hunk 2]
```diff
@@ -333,3 +333,36 @@ Once a endpoint is deleted, its tag can be reused.
 
   serve.delete_endpoint("simple_endpoint")
 
+How do I call an endpoint from Python code?
+-------------------------------------------
+
+To do so, you need to get a "handle" to that endpoint. To do so,
```
Comment (edoakes) [inline doc/source/serve/advanced.rst:339]: Nit: `To do so` repeated in both sentences

**Review by edoakes — APPROVED** [review]

File: doc/source/serve/advanced.rst (hunk 2 above)
Comment (anabranch) [reply doc/source/serve/advanced.rst]: thank you, fixed.

**Comment by edoakes** [comment]
@anabranch lint is failing: https://travis-ci.com/github/ray-project/ray/jobs/363843488#L696

**Comment by simon-mo** [comment]
uh oh: https://travis-ci.com/github/ray-project/ray/jobs/364683989#L699

**Comment by anabranch** [comment]
ok, I just simplified it - hopefully this clears it.

**Pull request merged**